### PR TITLE
Updates in readme documentation

### PR DESCRIPTION
**Pull request by shimilgithub**
- #741

### CODE REVIEW

**Review by AthulyaMS — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+</h1>
+<h4 align="center">Unified API for <a href="https://api.vachanengine.org/" target="_blank">Vachan-Engine applications</a></h4>
+<p align="center">
+  •<a href="#implementation-details"> Implementation Details</a> •
+  <a href="#start-app-with-docker">Start App</a> •
+  <a href="#set-up-locally-for-development-and-testingwithout-docker">Setup</a> •
+  <a href="#license">License</a>
+</p>
 
-Currently serving 3 client applications, VachanOnline website, the Vachan Mobile App and AutographaMT Bible translation tool.
```
Comment (Joel-Joseph-George) [inline README.md:5]: What about intellisgn and vachanadmin? do they come inside the specified projects?

Comment (shimilgithub) [reply README.md:5]: this is actually removed line(highlighted in red)

Comment (Joel-Joseph-George) [reply README.md:5]: Ohh sorry. I was viewing it in md view. So missed it.

**Review by Joel-Joseph-George — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+  Vachan API
+  <br>
+</h1>
+<h4 align="center">Unified API for <a href="https://api.vachanengine.org/" target="_blank">Vachan-Engine applications</a></h4>
+<p align="center">
+  •<a href="#implementation-details"> Implementation Details</a> •
+  <a href="#start-app-with-docker">Start App</a> •
+  <a href="#set-up-locally-for-development-and-testingwithout-docker">Setup</a> •
+  <a href="#license">License</a>
+</p>
```
Comment (kavitharaju) [inline README.md]: Markdown format is often used for its simplicity. I thinks its better to stick to it and avoid embedded HTML as much as possible.
In these we can use markdown features to get decent level of formatting
```
# big heading
## small heading
### smaller heading
[link-text](link-target)
- list item1
- list item2
   - sub item1
   - sub item2
```
# big heading
## small heading
### smaller heading
[link-text](#)
- list item1
- list item2
   - sub item1
   - sub item2

**Review by kavitharaju — COMMENTED** [review]
Regarding the content I am not cent percent sure. may be run it though Joel Mathew also once after merging

File: README.md (hunk 2 above)
Comment (shimilgithub) [reply README.md]: @kavitharaju Markdown doesn't directly  provide the option to align text and resize images. HTML is used for this purpose.

Comment (kavitharaju) [reply README.md]: How critical is aligning text in a readme?
About image resizing, yes you could use it places where markdown support is not there. But keep it limited to that

Comment (kavitharaju) [reply README.md]: Another thing is I am reviewing this PR with little context. The parent issue says
> Enhance the Readme file of Vachan API by providing comprehensive and clear documentation.

Which in my opinion doesn't say anything! What are we making more comprehensive and clear here? The issue itself should be more specific and to the point than a too generalized statement like this. 

And honestly I doubt the changes in the PR is making anything more comprehensive and clear.

Comment (kavitharaju) [reply README.md]: Something I liked here is the idea of adding a gif. Makes the document more interesting. But is the API docs the best thing we can show in it? May be... I dont know..

Comment (shimilgithub) [reply README.md]: > How critical is aligning text in a readme? About image resizing, yes you could use it places where markdown support is not there. But keep it limited to that

Text alignment is required only at the single line description("Unified API for VE applications") and the direct links mentioned below that line.

Comment (kavitharaju) [reply README.md]: Does it really have to be center? Why?

Comment (shimilgithub) [reply README.md]: to keep it properly down to the logo(which is kept in middle)

Comment (shimilgithub) [reply README.md]: > Another thing is I am reviewing this PR with little context. The parent issue says
> 
> > Enhance the Readme file of Vachan API by providing comprehensive and clear documentation.
> 
> Which in my opinion doesn't say anything! What are we making more comprehensive and clear here? The issue itself should be more specific and to the point than a too generalized statement like this.
> 
> And honestly I doubt the changes in the PR is making anything more comprehensive and clear.

will modify issue description

File: README.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
-
 <h1 align="center">
   <br>
   <a href="http://www.amitmerchant.com/electron-markdownify"><img src="https://api.vachanengine.org/static/images/Logo.svg" alt="vachanapi" width="200"></a>
   <br>
   Vachan API
   <br>
 </h1>
```
Comment (kavitharaju) [inline README.md:7]: Can't h1 , br be avoided?

Comment (shimilgithub) [reply README.md:7]: When I remove `h1` tag, logo image goes to left like this. Can I keep like that only or can I use `div` container to keep image centrally aligned.
Screenshot from 2023-11-06 13-40-04

Comment (kavitharaju) [reply README.md:7]: How about not keeping it centrally aligned? And I dont know what is the point of this much care for **formatting** a readme file...
```
<img src="https://api.vachanengine.org/static/images/Logo.svg" alt="vachanapi" width="100"></a>
# VACHAN API
```
vachanapi</a>
# VACHAN API

Comment (kavitharaju) [reply README.md:7]: And is <a href="http://www.amitmerchant.com/electron-markdownify"> needed?

Comment (shimilgithub) [reply README.md:7]: I have asked to update readme similar to this [reference](https://github.com/amitmerchant1990/electron-markdownify#readme)

Comment (shimilgithub) [reply README.md:7]: I have asked to update readme similar to this [reference](https://github.com/amitmerchant1990/electron-markdownify#readme)

**Pull request closed without merging**